worker/src/router.rs

```diff
@@ -66,6 +66,7 @@ impl<D> RouteContext<D> {
 
     /// Get the `Env` for this Worker. Typically users should opt for the `secret`, `var`, `kv` and
     /// `durable_object` methods on the `RouteContext` instead.
+    #[deprecated(since = "0.0.8", note="please use the `env` field directly")]
```

Comment:
The version should probably be `0.0.9` as this correct for `0.0.8` and the formatting for the `note` argument isn't correct.

Reply:
We can probably keep it as `0.0.8`, unless you specifically want to wait to include this in a future release (not the upcoming `0.0.8`)?

Reply:
Woops! I misremembered what version we were on. Never mind the version then.